scripts/start_docker_build_env.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck >/dev/null && shellcheck "$0"
+
+dir=$(pwd)
+parentdir="$(dirname "$dir")"
+export PYTH_REPO=$parentdir
+
+# Use the latest pythd image from https://hub.docker.com/r/pythfoundation/pyth-client/tags?name=pythd
+LATEST_VERSION="v2.17.3"
+export IMAGE=pythfoundation/pyth-client:pythd-${LATEST_VERSION}
+
+docker run -it \
+        --volume "${HOME}:/home/pyth/home" \
```

Comment:
It feels a bit excessive to mount the whole home dir. Is this intentational

Reply:
Thanks, i removed this. It was copied from the "Development Setup Using VSCode" section in the README. i don't think it's necessary if the only purpose is to be able to build the oracle program from the mounted directories.